menas/ui/service/EntityService.js

```diff
@@ -280,8 +280,28 @@ class DatasetService extends EntityService {
     return super.getByNameAndVersion(sName, iVersion, sModelPath, "#/dataset")
   }
 
+  cleanupEntity(oEntity) {
+    return {
+      description: oEntity.description,
+      name: oEntity.name,
+      schemaName: oEntity.schemaName,
+      schemaVersion: oEntity.schemaVersion,
+      hdfsPath: oEntity.hdfsPath,
+      hdfsPublishPath: oEntity.hdfsPublishPath,
+      title: oEntity.title,
+      version: oEntity.version,
+      conformance: ( oEntity.conformance || [] )
+    }
+  }
+
+  create(oDataset) {
+    let cleanDataset = this.cleanupEntity(oDataset);
```

Comment:
Why not add line to the `super.create` function? This will reduce duplication and remove the number of overriden functions that simply delegate.
This applies for `super.update` as well in all entities.